Library/Homebrew/dev-cmd/bump-formula-pr.rb

```diff
@@ -0,0 +1,73 @@
+require "formula"
+
+module Homebrew
+  def bump_formula_pr
+    formula_arg = ARGV.named.first
+    odie "No formula specified!" unless formula_arg
+    formula = Formula[ARGV.named.first]
+    odie "No #{formula} formula found!" unless formula
```

Comment:
Shouldn't this be `odie "No #{formula_arg} formula found!" unless formula` or `odie "No #{ARGV.named.first} formula found!" unless formula` instead, since `formula` will be `nil` in this case, and `formula_arg` will have the name string?